core/tokens.md

```diff
@@ -59,10 +59,20 @@ dto TokenIdentifierParts:
 
 ## TokenTransfer
 
+This is a synonym for `TokenAmount`.
+
+```
+dto TokenTransfer implements ITokenAmount:
+    token: Token;
+    amount: Amount;
+```
+
+## ITokenAmount
+
 ```
-dto TokenTransfer:
+interface ITokenAmount:
```

Comment:
This way, objects returned by `network_provider.get_tokens_of_account()` are compatible with the transactions factories.